bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -137,10 +137,12 @@ The controller bridge has the following channels:
 | colorTemperatureAbs | Number    | Color temperature (in Kelvin, 1200 to 6500) of all light panels                                           | No        |
 | colorMode           | String    | Color mode of the light panels                                                                            | Yes       |
 | effect              | String    | Selected effect of the light panels                                                                       | No        |
+| layout              | Image     | Shows the layout of your panels with IDs.                                                                 | Yes       |
```

Comment:
The heading for this column is "Read Only"

I believe Yes here is correct. It is just an image of the current layout. The user can't paint on the image and expect the panels to move on the wall. But e.g. for on/off state I can both see what it currently is (read), and change the state of it to e.g. turn it off (write).

Reply:
Ah, sorry, yes, you are right - my bad. This is the OUTPUT channel - I should have looked at the type. It read "shows" which told "me" it was a switch with which I can choose. Maybe it is better say "provides" instead of "shows"?